pkgs/applications/networking/p2p/btpd/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, openssl }:
+
+stdenv.mkDerivation rec {
+  pname = "btpd";
+  version = "0.16";
+
+  src = fetchFromGitHub {
+    owner = "btpd";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0r8jml306jjfr0cpmgjv78kjln0pq899vj1v3pgnys2j6i3wq9s0";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ openssl ];
```

Comment:
It seems to build with `openssl_1_1` as well, so I suggest to use that instead, as eventually [it should become the default](https://github.com/NixOS/nixpkgs/pull/22357).